### PR TITLE
Update content repo default branch

### PR DESCRIPTION
**Pull request by c0ldcalamari**

### CODE REVIEW

File: bakery/src/scripts/fetch_book_group.sh [hunk 1]
```diff
@@ -2,7 +2,7 @@
 exec > >(tee "${COMMON_LOG_DIR}"/log >&2) 2>&1
 
 reference=$(cat "${BOOK_INPUT}"/version)
-[[ "$reference" = latest ]] && reference=master
+[[ "$reference" = latest ]] && reference=main
```
Comment (rnathuji) [inline bakery/src/scripts/fetch_book_group.sh:5]: Based on this change it looks like we'll now adopt the convention / assumption that the default branch will be `main` for content repos, is that correct? If so, should / will we update `ce-git-storage-spike` accordingly?

Comment (rnathuji) [reply bakery/src/scripts/fetch_book_group.sh:5]: Based on Slack convos, it sounds like we're going with `main` as the default branch for all content repos.

**Review by rnathuji — APPROVED** [review]
Note: There's an unrelated CI failure which I believe is because of [this build update](https://concourse-v6.openstax.org/teams/CE/pipelines/qa-build-push-selenium/jobs/build-and-publish-images/builds/16).

File: bakery/src/scripts/fetch_book_group.sh (hunk 1 above)
Comment (c0ldcalamari) [reply bakery/src/scripts/fetch_book_group.sh:5]: [Resource ](https://stevenmortimer.com/5-steps-to-change-github-default-branch-from-master-to-main/)I used to update `ce-git-storage-spike`:


```
ce-git-storage-spike[master] $ git status
On branch master
Your branch is up to date with 'origin/master'.

nothing to commit, working tree clean
ce-git-storage-spike[master] $ git branch -m master main
ce-git-storage-spike[main] $ git push -u origin main
Total 0 (delta 0), reused 0 (delta 0)
remote: 
remote: Create a pull request for 'main' on GitHub by visiting:
remote:      https://github.com/openstax/ce-git-storage-spike/pull/new/main
remote: 
To github.com:openstax/ce-git-storage-spike.git
 * [new branch]      main -> main
Branch 'main' set up to track remote branch 'main' from 'origin'.
ce-git-storage-spike[main] $ git symbolic-ref refs/remotes/origin/HEAD refs/remotes/origin/main
ce-git-storage-spike[main] $ git push origin --delete master
To github.com:openstax/ce-git-storage-spike.git
 - [deleted]         master
ce-git-storage-spike[main] $ git status
On branch main
Your branch is up to date with 'origin/main'.

nothing to commit, working tree clean
```

with: https://docs.github.com/en/github/administering-a-repository/changing-the-default-branch

**Comment by c0ldcalamari** [comment]
How to test (for @omehes):

**Given** a book from below repo to CORGI for either GIT type jobs (PDF/Web-Preview):
https://github.com/openstax/ce-git-storage-spike
**Then** it should produce a successful build.

**Given** a book from below repo to CORGI for either GIT type jobs (PDF/Web-Preview):
https://github.com/openstax/college-algebra-bundle
**Then** it should produce a successful build.

**Comment by c0ldcalamari** [comment]
**Merged/Done:**
Deploy staging Corgi

**Ready for Testing:**
Otto picks when ready

**QA Testing:**
Otto tests on CORGI staging
If Otto tests pass, promote staging CORGI to production CORGI

**Comment by omehes** [comment]
testing on corgi-staging:

1.
Given a book from below repo to CORGI for either GIT type jobs (PDF/Web-Preview):
https://github.com/openstax/college-algebra-bundle

college-algebra-bundle/precalculus
college-algebra-bundle/precalculus-coreq - ok
college-algebra-bundle/college-algebra
college-algebra-bundle/algebra-and-trigonometry

Then it should produce a successful build.



2.
Given a book from below repo to CORGI for either GIT type jobs (PDF/Web-Preview):
https://github.com/openstax/ce-git-storage-spike
Then it should produce a successful build.

**Comment by omehes** [comment]
testing on corgi prod:

Given a book from below repo to CORGI for either GIT type jobs (PDF/Web-Preview):
https://github.com/openstax/college-algebra-bundle

college-algebra-bundle/precalculus - 
college-algebra-bundle/precalculus-coreq - 

Then it should produce a successful build.


Given a book from below repo to CORGI for either GIT type jobs (PDF/Web-Preview):
https://github.com/openstax/ce-git-storage-spike
Then it should produce a successful build.

ce-git-storage-spike/precalculus (col11667) - 
ce-git-storage-spike/college-algebra - 

Then it should produce a successful build.


Compared pdfs from Alg-trig from the above 2 repo jobs (used diffpdf.app): some differences (content not synced in ce-git-storage...
Compared these pdfs to legacy-staging created pdf and corgi pdf (archive) server: staging pdfs - the same discrepancy (out of sync books in legacy-staging and corgi pdf)
Compared these pdfs to legacy-prod created pdf and corgi pdf (archive) server: prod pdfs - same

Above issues are due to content not synchronised! All looks good

**Pull request merged**